vulnerablecode/settings.py

```diff
@@ -160,5 +160,6 @@
         "rest_framework.filters.SearchFilter",
     ),
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
-    "PAGE_SIZE": 100,
+    # Reduce to 10 records per page to display data almost instantaneously https://github.com/nexB/vulnerablecode/issues/819
```

Comment:
> almost instantaneously

This may not always be true, there are cases where displaying 10 records is not instantaneous.

Maybe something along:
Limit the load on the Database returning a small number of records by default.